scss/atoms/tables/_tables.scss

```diff
@@ -0,0 +1,95 @@
+/*
+* ==========================================================================
+* Tables
+* ==========================================================================
+*/
+
+@import
+	"dist/chassis",
+	"mixins";
+
+table {
+	width: 100%;
+	margin: map-get($table-base, margin);
+	font-size: map-get($table-base, font-size);
+	text-align: left;
+}
+
+thead {
+	border-bottom: map-get($table-base, border);
+	> tr > th {
+		border-top: 0;
+	}
+}
+
+td {
+	border-top: map-get($table-base, border);
+	padding: map-get($table-base, padding);
+}
+
+th {
+	border-top: map-get($table-base, border);
+	padding: map-get($table-base, padding);
+	color: map-get($table-base, color);
+	font-weight: 400;
+	font-size: map-get($table-base, thead-font-size);
+	white-space: nowrap;
+}
+
+table {
+	.active {
+		background-color: #f5f5f5;
+	}
+	.success {
+		background-color: #dff0d8;
+	}
+	.warning {
+		background-color: #fcf8e3;
+	}
+	.danger {
+		background-color: #f2dede;
+	}
+	.info {
+		background-color: #d9edf7;
+	}
+}
+
+.striped-table > tbody > tr:nth-of-type(odd), {
+	background-color: #f2f2f2;
+}
+
+.table--responsive {
+	position: relative;
+	z-index: 1;
+	margin-bottom: em( 20px );
+	border: 0;
+	background: #fff no-repeat;
+	background-image: radial-gradient( farthest-side at 0 50%,rgba( 0, 0, 0, .2 ), transparent ), radial-gradient( farthest-side at 100% 50%, rgba( 0, 0, 0, .2 ), transparent );
```

Comment:
get this from a mixin if possible, so we can use it in other components in future